Dump out stats for each iface

bowei opened the pull request.

bowei committed:
Dump out stats for each iface

bobbypage commented on run.sh:
```diff
@@ -127,6 +127,13 @@ while true; do
+  # ethtool for every interface
+  for iface in $(ip link | awk '/^[0-9]+: / {print $2}' | sed 's/:$//' | sed 's/@.*$//'); do
+    f=ethtool_${iface}
+    c=$(ethtool -S ${iface})
```
is this ok running within container or will being in network namespace be ok?

karan replied:
with host network it should be fine

bowei replied:
I assuming dumping the host side of the veth is sufficient
We can poke into each NS for completeness

bowei committed:
Add in net/snmp and zoneinfo proc files

- net/snmp is more stable than netstat -s
- zoneinfo is info on the memroy allocation
- Add ps output
- Make top output wide

bobbypage commented:
LGTM.
